Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -210,69 +217,70 @@ extension PackageGraph {
 /// commands as well as any diagnostics or output emitted by the plugin.
 public struct PluginInvocationResult {
     /// The plugin that produced the results.
-    public let plugin: PluginTarget
+    public var plugin: PluginTarget
     
     /// The commands generated by the plugin (in order).
-    public let commands: [Command]
+    public var commands: [Command]
```

Comment:
question: why move to `var`? testing?

Reply:
This seemed in line with other structs, where the fields would in general be `var` even if the struct is usually used as a `let`.  Otherwise, if the struct _is_ used as a `var`, the `let` fields still can't be changed, right?  So unless certain fields should absolutely only be set on initialization, it seemed more in line with other structs to make these `var`.  I could certainly change it back but feel like there's something to learn here.

Reply:
to be clear there is nothing wrong with making them `var`, my question was on the change from `let` to `var`: its often the case we choose `let` when we want to provide readonly access as the state is tightly managed internally, so changing to `var` raises the question if the code needs to be adjusted to external state changes

Reply:
I see.  In this case it was because it seemed to me that I had made a mistake in originally making them `let`, since would prevent building up a struct piecemeal.  In this case there is no code that depends on it yet, I was just trying to normalize things.  In this case it's a plain struct.